### DISCUSSION TITLE
When is model re-fetched?

### DISCUSSION THREAD

**Discussion by jchretien** [blakeblackshear/frigate#12784] [Ask A Question] [answered]
### What is your question:

I am wondering after we submitted images to Frigate+, when the model information is updated locally?

I thought by looking at the modification date on the files under config/model_cache the timestamp would change... But it's still at the creation date.

I thought the model is improving on most things, but for one particular thing it still makes lots of false positives so I'm curious if the model is actually updated.

Cheers!

**Comment by NickM-27** [comment, marked as answer]
You have to generate a new model and update the id in your config. Models are not trained automatically. 

https://docs.frigate.video/plus/first_model

**Reply by jchretien** [reply]
Ah! Now I understand why there are 12 models available... Thanks for answering I was completely unaware of this and unnecessarily sending hundreds of pictures.
